src/visualization/types/SimpleTable/view.tsx

```diff
@@ -8,27 +8,23 @@ import {FluxResult} from 'src/types/flows'
 import {VisualizationProps} from 'src/visualization'
 
 // Utils
-import {parseFromFluxResults} from 'src/timeMachine/utils/rawFluxDataTable'
-
-import './style.scss'
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 
+// Components
 import PageControl from 'src/visualization/types/SimpleTable/PageControl'
 import PagedTable from 'src/visualization/types/SimpleTable/PagedTable'
 import {PaginationProvider} from 'src/visualization/context/pagination'
 
+import './style.scss'
```

Comment:
We might consider renaming this from `style.scss` (search the codebase for `style.scss` and see how many there are) to something like `SimpleTableStyle.scss`, and making its path absolute rather than relative. It'll be much easier to work with if we can search for import path, rather than just `style.scss`.

Reply:
Done.